test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test

```diff
@@ -160,12 +160,15 @@ Date: REGEX(.*)
 </contextAttributeResponse>
 2: ++++++++++++++++++++
 HTTP/1.1 200 OK
-Content-Length: 685
+Content-Length: 755
 Content-Type: application/xml
 Date: REGEX(.*)
 
 <?xml version="1.0"?>
 <appendContextElementResponse>
+  <entityId type="" isPattern="false">
```

Comment:
Well, we decided we wanted to add the info of the entity for AppendContextElementResponse when used ... (I don't remember exactly why) ... with GET /v1/contextEntities ... No, no me acuerdo